### ISSUE TITLE
ANMN_MHLWAVE warnings in log

### ISSUE THREAD

**Issue by mhidas** [aodn/harvesters#195]
Same warnings as in #194, suggesting incorrect context load setup?

> begin;2015-03-29 20:36:05+11:00
> Info: Parameter "Destination_Database" is a new parameter of Implicit_Context_Context
> Info: Parameter "Destination_Port" is a new parameter of Implicit_Context_Context
> Info: Parameter "Destination_Server" is a new parameter of Implicit_Context_Context
> Info: Parameter "Destination_Schema" is a new parameter of Implicit_Context_Context
> Info: Parameter "Destination_Login" is a new parameter of Implicit_Context_Context
> Info: Parameter "Destination_Password" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_Username" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_Password" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_URL" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_SpatialTable" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_SpatialColumn" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_SpatialResolution" is a new parameter of Implicit_Context_Context
> Info: Parameter "MHLwaveMetadata_File" is a new parameter of Implicit_Context_Context
> Info: Parameter "Metadata_UUID" is a new parameter of Implicit_Context_Context
> Info: Parameter "include" is a new parameter of Implicit_Context_Context
> Info: Parameter "Threddscatalog" is a new parameter of Implicit_Context_Context

**Comment by bpasquer** [comment]
I checked the harvester . The context load is correctly set . I found one error though, hence the PR i made, but i doubt it would be the cause of this warning

**Comment by mhidas** [comment]
Thanks @bpasquer . I merged your PR, but I agree that's unrelated. After a quick look, I've just realised one reason this is appearing in this log and not all others may simply be that we normally disable all info/warning messages of this kind (see [Context override](https://docs.google.com/document/d/1-E05wDMvI_Yy2Zz9m3D8DCWUjjkhPDJFm_jr9TvbUgc/edit?pli=1#heading=h.5eucxwr7xoc4) section in Google doc).

However, I also noticed you have the implicit context load enabled for some of the subjobs. These should all be disabled, as the subjobs get their context values from the main job (that's why we use the "transmit whole context" option).
